pkg/process/checks/remote_sysprobe_util_test.go

```diff
@@ -0,0 +1,103 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package checks
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+
+	"github.com/DataDog/agent-payload/v5/process"
+	netMocks "github.com/DataDog/datadog-agent/pkg/process/net/mocks"
+	"github.com/DataDog/datadog-agent/pkg/process/procutil"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func makeProcStatsWithPerm(pid int32) *process.ProcStatsWithPerm {
+	r := rand.New(rand.NewSource(int64(pid)))
+	return &process.ProcStatsWithPerm{
+		OpenFDCount: r.Int31(),
+		ReadCount:   r.Int63(),
+		WriteCount:  r.Int63(),
+		ReadBytes:   r.Int63(),
+	}
+}
+
+func TestMergeProcWithSysprobeStats(t *testing.T) {
+	assertMatchesSysProbeStats := func(t *testing.T, proc *procutil.Process, stats *process.ProcStatsWithPerm) {
+		assert.Equal(t, stats.OpenFDCount, proc.Stats.OpenFdCount)
+		assert.Equal(t, stats.ReadCount, proc.Stats.IOStat.ReadCount)
+		assert.Equal(t, stats.WriteCount, proc.Stats.IOStat.WriteCount)
+		assert.Equal(t, stats.ReadBytes, proc.Stats.IOStat.ReadBytes)
+		assert.Equal(t, stats.WriteBytes, proc.Stats.IOStat.WriteBytes)
+	}
+	hasSysProbeStats := func(proc *procutil.Process) bool {
+		return proc.Stats.IOStat.IsZeroValue() && proc.Stats.OpenFdCount != 0
```

Comment:
```suggestion
		return !proc.Stats.IOStat.IsZeroValue() && proc.Stats.OpenFdCount != 0
```
We should be checking that it's _not_ the zero value, right?

Reply:
Ah yes, that's correct thank you